.github/workflows/dco.yml

```
@@ -16,7 +16,7 @@ jobs:
           (github.event.comment.body == 'recheck' || github.event.comment.body
           == 'I have read the DCO document and I hereby sign the DCO.') ||
           github.event_name == 'pull_request_target'
-        uses: cla-assistant/github-action@v2.3.0
+        uses: cla-assistant/github-action@v2.6.1
```

```suggestion
        uses: cla-assistant/github-action@ca4a40a7d1004f18d9960b404b97e5f30a505a08 # v2.6.1
```

We should update this to use the SHA.

---

is there a way to configure renovate to use the sha in this instance? 🤔

---

I think renovate automatically uses it if we use it. I think they follow whatever we have in place.